libs/parsers/src/lib/expression/parser.ts

```diff
@@ -62,14 +69,14 @@ export class ExpressionParser {
           value: expression.substring(1),
         };
       } else {
-        return ExpressionParser.parseExpression(expression.substring(1));
+        return ExpressionParser.parseExpression(expression.substring(1), isAliasPathParsingEnabled);
       }
     } else {
       return ExpressionParser._parseStringInterpolationExpression(expression);
     }
   }
 
-  private static _parseExpressionRecursively(scanner: ExpressionScanner, index = 0): Expression {
+  private static _parseExpressionRecursively(scanner: ExpressionScanner, index = 0, isAliasPathParsingEnabled?: boolean): Expression {
```

Comment:
Better to keep this argument isAliasPathParsingEnbaled as required in all private functions so we can make sure it is getting passed down from all callers, else some code paths can be missed.